framework/src/auxkernels/AuxKernel.C

```diff
@@ -78,6 +78,7 @@ AuxKernel::AuxKernel(const InputParameters & parameters) :
     _assembly(_subproblem.assembly(_tid)),
 
     _var(_aux_sys.getVariable(_tid, parameters.get<AuxVariableName>("variable"))),
+    _displaced_var(parameters.getCheckedPointerParam<FEProblem *>("_fe_problem")->getDisplacedProblem() ? &_sys.getVariable(_tid, parameters.get<AuxVariableName>("variable")) : NULL),
```

Comment:
If the displaced problem exists - then just grab the displaced AuxSystem from it here...